### DISCUSSION TITLE
Suggest new Flash games to add to the Vanadium Arcade here.

### DISCUSSION THREAD

**Discussion by Null-n-V0id** [titaniumnetwork-dev/Vanadium#16] [Ideas]
Arcades are an important part of an unblocker website, so keeping them stocked with games is a huge priority.

Requirements for a game to be added:

-Full, or nearly full compatibility with the Ruffle.rs emulation layer.
-No sexual content, shock content, etc.
-Game must not exceed 50 MB in file size.

I will manually review all submissions, and any final decisions will be made at my discretion.

Feel free to include a pull request with your suggestion, if you'd like.

**Comment by jangodev** [comment]
List of flash games to add:
- Grow Cube
- Dino Run
- Alien Hominid
- Warfare 1917
- Earn to die 2
- Hatoful Boyfriend

**Reply by Null-n-V0id** [reply]
Can you provide links to these games, and verify whether they are operational with Ruffle.rs? I'll be looking into these titles shortly.

**Reply by Null-n-V0id** [reply]
Results of testing:

Grow Cube - Not Adding (Reason: Ruffle.rs incompatibility)
Dino Run - Not Adding (Reason: Ruffle.rs incompatibility)
Alien Hominid - Adding (Will be added in next commit)
Warfare 1917 - Adding (Will be added in next commit)
Earn to Die 2 Exodus - Not Adding (Reason: Ruffle.rs incompatibility)
Hatoful Boyfriend - Not Adding (Reason: Not an Adobe Flash game)

All compatible titles will be added to Vanadium in the next commit.

**Comment by FLOOFYBULL** [comment]
• Super Mario Flash

•Super Mario Flash 2

•Super Mario Flash 3

**Reply by Null-n-V0id** [reply]
Can you provide links to these games, and verify whether they are operational with Ruffle.rs? I'll be looking into these titles shortly.

**Reply by Null-n-V0id** [reply]
Results of testing:

Super Mario Flash - Adding (Will be added in next commit)
Super Mario Flash 2 - Not Adding (Reason: Ruffle.rs incompatibility)
Super Mario Flash 3 - Not Adding (Reason: Ruffle.rs incompatibility)

All compatible titles will be added to Vanadium in the next commit.

**Reply by FLOOFYBULL** [reply]
Nice, too bad super mario flash 2 and 3 don't work, but still good to know super mario flash will be on Vanadium

**Reply by Null-n-V0id** [reply]
yep. I added Super Mario Flash in the previous commit, so anyone who deploys Vanadium after yesterday will have it included on their website.

**Comment by whoshockyy** [comment]
ay cuh burrito bison cuh no kizzy cuh por favor cuh

**Reply by Null-n-V0id** [reply]
what

**Comment by average-game-enjoyer** [comment]
new grounds rumble 
age of war 
super meat boy

**Reply by Null-n-V0id** [reply]
Can you provide links to these games, and verify whether they are operational with Ruffle.rs? I'll be looking into these titles shortly.

**Reply by average-game-enjoyer** [reply]
newgrounds rumble-https://www.newgrounds.com/portal/view/381115, and yes 
age of war- https://armorgames.com/play/616/age-of-war, and yes 
meat boy-https://www.newgrounds.com/portal/view/463241, and yes

**Reply by Null-n-V0id** [reply]
Results of testing:

Age of War - Adding (Will be added in next commit)
Newgrounds Rumble - Adding (Will be added in next commit)
Super Meat Boy - Adding (Will be added in next commit)

All compatible titles will be added to Vanadium in the next commit.

**Comment by Null-n-V0id** [comment]
FYI: This repository is open to pull requests if you wish to add your own tested games!

**Comment by pbkx** [comment]
Ruffle compatibility is horrendous, are there any other options? Most decent semi-new flash games have no support.

Also, you can probably optionally improve the speed of some games by storing the swf files in the assets folder.

**Reply by Null-n-V0id** [reply]
Unfortunately Ruffle is the best we got, but I will be redirecting the games to a local directory if/when I get around to updating the proxy again.
